package.json

```diff
@@ -48,7 +48,6 @@
     "@ohdsi/visibilityjs": "^2.0.2",
     "ajv": "^6.10.0",
     "bootstrap": "^3.3.7",
-    "bootstrap-datetimepicker": "^0.0.7",
```

Comment:
What was the reason to switch from npm-hosted bootstrap-datetimepicker to a local copy?